code/src/components/HappyThoughtCards.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react'
+import ThoughtForm from './ThoughtForm'
+import { formatDistance } from 'date-fns'
+
+// Creating HappyThoughtsCards, fetching thoughts data
+const HappyThoughtCards = ({thought}) => {
+    const [thoughts, setThoughts] = useState([])
+
+    useEffect (() => {
+        fetch("https://happy-thoughts-technigo.herokuapp.com/thoughts")
+        .then(res => res.json())
+        .then(thoughts => setThoughts(thoughts))
+}, [])
+
+
+// Post likes to the API
+const onLike = (thoughtId) => {
+    fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${thoughtId}/like`, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json'},
+      body: ''
```

Comment:
you may not need line 21 if the request does not have any body data ! :)

Reply:
Thank you, had no idea! 💯